opm/autodiff/SimulatorFullyImplicitBlackoilEbos.hpp

```diff
@@ -157,24 +157,31 @@ class SimulatorFullyImplicitBlackoilEbos
         std::string tstep_filename = output_writer_.outputDirectory() + "/step_timing.txt";
         std::ofstream tstep_os(tstep_filename.c_str());
 
+        const auto& schedule = eclState().getSchedule();
+
         // adaptive time stepping
         std::unique_ptr< AdaptiveTimeStepping > adaptiveTimeStepping;
         if( param_.getDefault("timestep.adaptive", true ) )
         {
-            adaptiveTimeStepping.reset( new AdaptiveTimeStepping( param_, terminal_output_ ) );
+
```

Comment:
Hu? The FIP caculations are connected to the settings of the adaptive time stepper? That is quite a surprise. Please explain!